azure-pipelines.yml

```diff
@@ -90,9 +112,33 @@ jobs:
       azureContainerRegistry: '{"loginServer":"gpitfuturesdevacr.azurecr.io", "id" : "/subscriptions/7b12a8a2-f06f-456f-b6f9-aa2d92e0b2ec/resourceGroups/gpitfutures-dev-rg-acr/providers/Microsoft.ContainerRegistry/registries/gpitfuturesdevacr"}'
       dockerComposeFile: '**/docker-compose.yml'
       action: 'Push services'
-      additionalImageTags: '$(Build.BuildNumber)'
+      additionalImageTags: '$(semVer)'
       includeSourceTags: true
 
+  - task: HelmInstaller@1
+    displayName: 'Install helm'
+    inputs:
+      helmVersionToInstall: 'latest'
+  - bash: |
+      cd charts
+      helm package \
+          --version $(semVer) \
+          --app-version $(semVer) \
+          admin
+    failOnStderr: true
+    displayName: 'Helm package'
+  - bash: |
+      cd charts
+      chartPackage=$(ls admin-*.tgz)
+      az acr helm push --force \
+          -n "gpitfuturesdevacr" \
+          -u "gpitfuturesdevacr" \
+          -p $(gpitfuturesdevacr-pass) \
+          $chartPackage      
+    failOnStderr: true
+    name: helmPush
+    displayName: 'Push helm to az acr'
+
   - task: DockerCompose@0
     displayName: 'Build: Test'
     condition: and(succeeded(), ne(variables['build.reason'], 'PullRequest'))
```

Comment:
can you add to this condition to make sure it's development or master only?

Reply:
Wouldn't that be redundant? Unless I'm missing something, the build pipeline gets triggered only on commit & PR to master || dev [more info here](https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#ci-triggers)

And by omitting this step when the reason is a PR, this step will only run on commits made to master || dev .

Unless the reason you'd like it to be there for manual triggers, I think its pointless.

 Please let me know what do you think.